src/utils/conversion.ts

```diff
@@ -4947,3 +4963,86 @@ export function confidentialTransactionIdToBigNumber(
 ): BigNumber {
   return new BigNumber(id.toString());
 }
+
+/**
+ * @hidden
+ */
+export function meshPublicKeyToKey(publicKey: PalletConfidentialAssetConfidentialAccount): string {
+  return publicKey.toString();
+}
+
+/**
+ * @hidden
+ */
+export function meshAssetAuditorToAssetAuditors(
+  rawAuditors: BTreeMap<U8aFixed, BTreeSet<PalletConfidentialAssetAuditorAccount>>,
+  context: Context
+): { assets: ConfidentialAsset[]; auditors: ConfidentialAccount[] } {
```

Comment:
The chain has the modeled with `Record<AssetId, BTreeSet<Audtior>`. 

Here I bring out the auditors to be siblings, which looses a bit of information, but makes for a simpler API. Users can still check normally required auditors for the assets returned.